src/MoreSpeakers.Web/Areas/Admin/Pages/Users/Index.cshtml

```diff
@@ -0,0 +1,167 @@
+@page
+@model MoreSpeakers.Web.Areas.Admin.Pages.Users.IndexModel
+@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
+
+@{
+    ViewData["Title"] = "Users";
+    var q = Model.Query;
+    int totalPages = (int)Math.Ceiling((double)Model.Result.TotalCount / Math.Max(1, Model.Result.PageSize));
+    totalPages = Math.Max(1, totalPages);
+
+    string SortDirFor(string col)
+    {
+        var isSame = string.Equals(q.Sort, col, StringComparison.OrdinalIgnoreCase);
+        var dir = isSame && string.Equals(q.Dir, "asc", StringComparison.OrdinalIgnoreCase) ? "desc" : "asc";
```

Comment:
These should not be hard coded but used as the description or name from the enum